atef/widgets/config/result_summary.py

```diff
@@ -0,0 +1,497 @@
+"""
+Widgets for summarizing the results of a run.
+"""
+from __future__ import annotations
+
+import csv
+import dataclasses
+from typing import Any, List, Set
+
+from qtpy import QtCore, QtGui, QtWidgets
+from qtpy.QtCore import Qt
+
+from atef.config import PreparedFile
+from atef.enums import Severity
+from atef.procedure import PreparedProcedureFile
+from atef.type_hints import AnyDataclass
+from atef.walk import walk_config_file, walk_procedure_file
+from atef.widgets.core import DesignerDisplay
+from atef.widgets.utils import insert_widget
+
+
+@dataclasses.dataclass
+class ResultInfo:
+    """ Normalized, and slightly processed view of configs/steps with results """
+    status: Severity
+    name: str
```

Comment:
`name` always seems to come from `origin` directly in the `from_file` classmethod below - could it be a property?